tests/create_all/main.tf

```diff
@@ -0,0 +1,31 @@
+module "state_bucket" {
+  source = "git::https://github.com/plus3it/terraform-aws-tardigrade-s3-bucket.git?ref=5.0.0"
+
+  bucket        = "test-bucket-for-ackend"
+  force_destroy = true
+  versioning    = "Enabled"
+
+  public_access_block = {
+    block_public_acls       = true
+    block_public_policy     = true
+    ignore_public_acls      = true
+    restrict_public_buckets = true
+  }
+
+  server_side_encryption_configuration = {
+    bucket_key_enabled = true
+    sse_algorithm      = "aws:kms"
+    kms_master_key_id  = null
+  }
+
+}
+
+resource "aws_dynamodb_table" "this" {
+  name                        = "test-table-for-backend"
+  deletion_protection_enabled = true
+  billing_mode                = "PAY_PER_REQUEST"
+}
+
+data "aws_caller_identity" "current" {}
+
+data "aws_partition" "current" {}
```

Comment:
Looks like these two data sources aren't being used, so they can be removed from the test.